### ISSUE TITLE
Project Proposal Peer Review

### ISSUE THREAD

**Issue by YijiaCui** [SZ559/Analysis-of-Learning-Rate-for-Deep-Neural-Networks#2]
This group’s project is about deep neural networks and methods to improve their adaptive learning rate. They plans to study and analyze the existing algorithms for adaptive learning rate for Deep Neural networks, and to come up with a new method to identify an optimal learning rate and to improve the performance of existing algorithm. After they finish improving the algorithms, they will use several dataset to validate the reliability of the improved algorithms. 

Three points that I like about this proposal:
1. It is very professionally written. 
2. They have done previous work for this project. They read paper about the topics, and both of them have experience working on neural networks.
3. They have clear timeline.

Three things that can be improved:
1. It would be better if they provide an example of dataset that they are going to use to test the reliability.
2. For the significance part, it would be better if they can provide some example of how improved neural network will perform better in different fields. The topic is awesome, but it can be a little hard to understand for people who have never heard of neural network.
3. This proposal is so well written that I cannot come up with the third point of improvement. But in general, more examples and explanations of the concepts will be helpful.
